user-script-register-v2/popup/user-script.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="user-script.css"/>
+  </head>
+
+<body>
+
+  <div id="outer-wrapper">
```

Comment:
I think you don't need this. It seems to be used only in the CSS:

```css
#outer-wrapper {
  padding: 0.5em;
}
```

but you can just do:

```css
body {
  padding: 0.5em;
}
```